[BUG] 500 error when saving Application Configuration without logo

### Description

On a fresh install of Paperless-ngx v2.18.0, when I change the Application Title in the Application Configuration and leave the logo empty, I get a 500 Server Error.

<img width="981" height="481" alt="Image" src="https://github.com/user-attachments/assets/62d0f5ae-6021-4b0d-81a9-ace508eaa99b" />

If I upload a logo image, it works. If I don’t upload one, it gives the error.

### Steps to reproduce

1. Fresh install of Paperless-ngx v2.18.0.
2. Go to Application Configuration.
3. Change the Application Title.
4. Leave the logo field empty.
5. Save → results in 500 error.

### Webserver logs

```bash
webserver-1  | AttributeError: 'NoneType' object has no attribute 'read'
webserver-1  | [2025-08-17 18:57:04,661] [ERROR] [django.request] Internal Server Error: /api/config/1/
webserver-1  | Traceback (most recent call last):
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 489, in thread_handler
webserver-1  |     raise exc_info[1]
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 42, in inner
webserver-1  |     response = await get_response(request)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 489, in thread_handler
webserver-1  |     raise exc_info[1]
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
webserver-1  |     response = await wrapped_callback(
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 439, in __call__
webserver-1  |     ret = await asyncio.shield(exec_coro)
webserver-1  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/current_thread_executor.py", line 40, in run
webserver-1  |     result = self.fn(*self.args, **self.kwargs)
webserver-1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 493, in thread_handler
webserver-1  |     return func(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
webserver-1  |     return view_func(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/rest_framework/viewsets.py", line 125, in view
webserver-1  |     return self.dispatch(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/rest_framework/views.py", line 515, in dispatch
webserver-1  |     response = self.handle_exception(exc)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/rest_framework/views.py", line 475, in handle_exception
webserver-1  |     self.raise_uncaught_exception(exc)
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/rest_framework/views.py", line 486, in raise_uncaught_exception
webserver-1  |     raise exc
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/rest_framework/views.py", line 512, in dispatch
webserver-1  |     response = handler(request, *args, **kwargs)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/rest_framework/mixins.py", line 82, in partial_update
webserver-1  |     return self.update(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/rest_framework/mixins.py", line 67, in update
webserver-1  |     serializer.is_valid(raise_exception=True)
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/rest_framework/serializers.py", line 225, in is_valid
webserver-1  |     self._validated_data = self.run_validation(self.initial_data)
webserver-1  |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/src/paperless/src/paperless/serialisers.py", line 204, in run_validation
webserver-1  |     return super().run_validation(data)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/rest_framework/serializers.py", line 444, in run_validation
webserver-1  |     value = self.to_internal_value(data)
webserver-1  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/rest_framework/serializers.py", line 503, in to_internal_value
webserver-1  |     validated_value = validate_method(validated_value)
webserver-1  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/src/paperless/src/paperless/serialisers.py", line 212, in validate_app_logo
webserver-1  |     if magic.from_buffer(file.read(2048), mime=True) == "image/svg+xml":
webserver-1  |                          ^^^^^^^^^
```

### Browser logs

```bash

```

### Paperless-ngx version

v.2.18.0

### Host OS

Debian 13

### Installation method

Docker - official image

### System status

```json

```

### Browser

Zen

### Configuration changes

_No response_

### Please confirm the following

- [x] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [x] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [x] I have already searched for relevant existing issues and discussions before opening this report.
- [x] I have updated the title field above with a concise description.

--- a/src/paperless/serialisers.py
+++ b/src/paperless/serialisers.py
@@ -209,7 +209,7 @@ class ApplicationConfigurationSerializer(serializers.ModelSerializer):
         return super().update(instance, validated_data)
 
     def validate_app_logo(self, file):
-        if magic.from_buffer(file.read(2048), mime=True) == "image/svg+xml":
+        if file and magic.from_buffer(file.read(2048), mime=True) == "image/svg+xml":
             reject_dangerous_svg(file)
         return file
 

